auction9-backend/handler.js

```diff
@@ -0,0 +1,34 @@
+// Require for mysql connection
+// Configuration is in serverless.yml file inside 'environment:'
+const mysql = require('serverless-mysql')({
+  config: {
+    host	: process.env.MYSQL_ENDPOINT,
+    port	: process.env.MYSQL_PORT,
+    database: process.env.MYSQL_DATABASE,
+    user	: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD
+  }
+});
+
+/* getAuction - will return selected auction from DB
+ * GET: /auction/id - we get auction by ID
+ */
+export const getAuction = async (event, context) => {
+	try {
+		try {
```

Comment:
Seems that we don't need nested try/catch?

Reply:
First try/catch removed